apps/alarm/app.js

```diff
@@ -9,8 +9,8 @@ module.exports = function (activity) {
   logger.log('alarm load')
   var AlarmCore = new Core(activity)
   activity.once('notification', (channel) => {
-    logger.log('alarm notification event')
-    if (channel === 'on-ready') {
+    logger.log('alarm notification event: ', channel)
+    if (channel === 'on-ready' || channel === 'on-system-booted') {
```

Comment:
Does it cause receive two notifications within one boot?

Reply:
Yes, the issue have been addressed [here](https://github.com/yodaos-project/yodart/pull/509#discussion_r260192693)

Reply:
What my concern is after received the two different notifications, there's same code invoked below?
In my opinion:
* when receive `on-system-booted`, initialize alarm. 
* And when receive `on-ready`, synchronize time.

Reply:
\cc @ynliSonia